packages/webawesome/docs/assets/styles/utils.css

```
@@ -79,6 +79,20 @@
       font-size: var(--wa-font-size-xs);
       --max-width: unset;
     }
+
+    &.shrink {
+      font-size: round(calc(var(--wa-font-size-2xs) / 1.125), 1px); /* ~10px */
+    }
+  }
+
+  /* planned badge */
+  wa-badge.planned {
+    background-color: var(--wa-color-neutral-fill-quiet);
```

```suggestion
    background-color: var(--wa-color-neutral-fill-quiet);
    color: var(--wa-color-neutral-on-quiet);
```

Just to make sure that we're hitting appropriate contrast ratios. We can't _necessarily_ guarantee that themes will be designed so that `var(--wa-color-*-on-normal)` (the default badge text) works with `various(--wa-color-*-fill-quiet)`, but we can use the intended pairing to be extra safe.